support/config/constants.go

```diff
@@ -61,7 +61,8 @@ const (
 	// management cluster's resource group in Azure.
 	AROHCPKeyVaultManagedIdentityClientID = "ARO_HCP_KEY_VAULT_USER_CLIENT_ID"
 
-	ManagedAzureCredentialsFilePath          = "MANAGED_AZURE_HCP_CREDENTIALS_FILE_PATH"
+	ManagedAzureCredentialsFilePath = "MANAGED_AZURE_HCP_CREDENTIALS_FILE_PATH"
+	// TODO - MIv3 - this release version check can be removed once 4.18 and 4.19 both support MIv3
```

Comment:
which check is this referring to?

Reply:
The next line down ManagedAzureClientIdEnvVarKey